website-frontend/src/routes/events/+page.svelte

```diff
@@ -0,0 +1,48 @@
+<script lang="ts">
+	import { Event, Events } from '$lib/models/event';
+	import FeaturedEventCard from '$lib/components/events/FeaturedEventCard.svelte';
+
+	/** @type {import('./$types').PageData} */
+	export let data;
+
+	$: ({ events } = data);
+	events?.sort((e0: Event, e1: Event) => {
```

Comment:
Again, to fix the sorting problem, simply add `$:` as prefix to this code line.

Although the above suggestion might solve the sorting issue, I recommend using `toSorted` as it creates a shallow copy of the `events` array and sorts it based on the defined callback function. This promotes a much consistent programming practice of assuming that the elements we work on should stay as is, and not modify the original array.

For instance, instead of:
```ts
$: events?.sort((e0: Event, e1: Event) => {
```
Use the following code suggestion instead:
```ts
$: let sorted_events = events?.toSorted((e0: Event, e1: Event) => {
```